pkgs/by-name/ma/mackup/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  procps,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "mackup";
+  version = "0.8.40";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "lra";
+    repo = "mackup";
+    rev = "refs/tags/${version}";
+    hash = "sha256-hAIl9nGFRaROlt764IZg4ejw+b1dpnYpiYq4CB9dJqQ=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    poetry-core
+    pythonRelaxDepsHook
+    setuptools
+    docopt
+    nose
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    setuptools
+    six
+    docopt
+  ];
+
+  pythonImportsCheck = [ "mackup" ];
+
+  checkPhase = "nosetests --with-coverage --cover-branches --cover-package=mackup";
+  postPatch = ''
+    substituteInPlace mackup/utils.py \
+      --replace-fail '"/usr/bin/pgrep"' '"${lib.getExe' procps "pgrep"}"'
+  '';
```

```suggestion
  checkPhase = ''
    nosetests
  '';
```
we don't care about coverage or even want to abort the build it is below XX